packages/logger/src/types/Log.ts

```diff
@@ -9,7 +9,7 @@ type LogLevelThresholds = {
   [key in LogLevel]: number;
 };
 
-type LogAttributeValue = string | number | boolean | null | undefined | LogAttributeValue[] | { [key: string]: LogAttributeValue } | Error;
+type LogAttributeValue = unknown;
```

Comment:
Question: what was the reason why you removed these more granular types?

Reply:
The reason is exactly this PR: relaxing the type restrictions. I could have added `| unknown` at the end of the union type, but that makes little sense IMHO since `unknown` just includes all the others.

Reply:
I don't like to be pushy when I can't propose a solution (like now), but is there any way that we can explore using [Generics](https://www.typescriptlang.org/docs/handbook/2/generics.html) in a similar way that we do for the Tracer [here](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/tracer/src/types/Tracer.ts#L32)?

In this case I'm afraid this is a bit above my typing skills so I can't suggest the actual type, but in that case we could at least try restricting types that we never want (i.e. function references, etc.).

Also for transparency, when we talked offline I know I said `unknown` could work. It was because at the time I had no better idea; now I think generics could work(?)